lib/cache.js

```diff
@@ -151,12 +151,12 @@ function updated(curr, { serviceName, item }, { idField, updatedAtField }) {
     }
   }
 
-  let next
+  let next = curr
```

Comment:
Do you think we should be initialising `next` to `{...curr}` to make sure we're never mutating `curr`?

It looks like `setIn` already makes a copy so it would be redundant right now, but also a bit safer

Reply:
I'd say it'd just be slightly wasteful in this case. In application code maybe - but in library code it's ok to "tune it up a bit". Wasteful in that setIn already does that and in some other parts of the code where we do next = curr we might not end up changing the next at all, so better to return same reference and avoid re-render.